grpc_server/mix.exs

```diff
@@ -0,0 +1,152 @@
+defmodule GRPC.Server.MixProject do
+  use Mix.Project
+
+  @source_url "https://github.com/elixir-grpc/grpc"
+  @version "1.0.0"
```

Comment:
let's not leave 1.0.0 in these modules. If any publishing ends up happening we'll lose this version on Hex